components/ZoomImage/ZoomImage.tsx

```diff
@@ -16,25 +21,23 @@ export const ZoomImage: FC<ImageProps> = (props) => {
 
   return (
     <>
-      {isModalOpen ? (
+      {isModalOpen && !isGoogleAvatar && (
```

Comment:
I think it is not the right way
my suggestion:
create a utility function to detect image url type (google, github, upper, telegram) with return type { type: google | github | ... , zoomable: boolean }
and use it with zoomable prop
by this we do not need isGoogleAvatar prop and it will be good for open-closed prinsiple

Reply:
Done